deprecated/README.md

```diff
@@ -0,0 +1,8 @@
+# Deprecated Deployments
+
+This folder contains deprecated versions of 1Password SCIM bridge deployments. 
+
+At the time of deprecation, these deployments are still fully functional, but may no longer be updated and will eventually be removed.
+
+## Deployments
+ - aws-terraform (2020-12-21)
```

Comment:
Can we turn this one to a link as well please

Reply:
```suggestion
 - [aws-terraform](./aws-terraform) (2020-12-21)
```